types/passport-local-mongoose/index.d.ts

```diff
@@ -1,119 +1,118 @@
-// Type definitions for passport-local-mongoose 4.0.0
+// Type definitions for passport-local-mongoose 6.1.0
```

Comment:
```suggestion
// Type definitions for passport-local-mongoose 6.1
```

Reply:
you could also drop `"dt-header": false,` exclusion from `tslint.json`

Reply:
done